source/door_b_enb.rst

```diff
@@ -49,38 +58,39 @@ What is the response?
   mechanisms (valves) can be sorted out later and are not in any sense urgent.
-* Ask the Lead Systems Engineer and the Flight Director to send the command 1MCMDBDS to
+* Recommend to the Lead Systems Engineer and the Flight Director to send the command 1MCMDBDS to
   disable the door mechanism, side B. They may wish to test the command link by first
   sending a no-op command to the SI RCTU (CNOOPSI command). Use ``CAP 1438`` as a guide.
   The side A command would be 1MCMDADS.
-* Contact the GOT Duty Officer to inform that we need the dump data as soon as possible and to
-  email or call us when the dump file is available.
-* Process the dump data and make sure that there is nothing anomalous in the data *BEFORE*
-  the shutdown. We want to know if a new occurrence looks just like the previous occurrences.
-  If yes, it should appear as if in one frame the door mechanism was enabled.
-* Once analysis of the dump data is complete, convene a telecon at the next reasonable moment
-  with the ACIS team and review the diagnosis. The MIT ACIS team (Peter Ford, Bob Goeke, Mark
-  Bautz, and Bev LaMarr) should also be included in the discussion, either in the telecon or
-  via email. If the diagnosis is consistent with previous anomalies, (TBR... proceed with the
-  recovery). If the diagnosis is not consistent with previous spurious command anomalies,
-  stop and start a more involved analysis with the ACIS team.
 
-...possibly useful things left over from the DEA-A poweroff page I cribbed from... TBR
 
-* Identify whether or not additional comm time is needed and if so ask the OC/LSE to request it.
-* Prepare a CAP and submit it for review to capreview AT ipa DOT harvard DOT edu, and cc: acisdude.
+If there is not time during the comm at which the anomaly was discovered to send the above commands: 
+
+* Prepare a CAP using ``CAP 1438`` as a guide to send the commands at the next
+  opportunity.  Submit it for review to capreview AT ipa DOT harvard DOT edu, and cc: acisdude.
   It will also be necessary to call the OC/CC to determine which number should be used for the CAP.
   This CAP will have the following steps:
 
-  - Send a NOP command, CNOOPSI, and monitor command count increment.
+  - Send a NO-OP command, CNOOPSI, and monitor command count increment.
   - Send command 1MCMDBDS to disable side B of the door mechanism. (Side A: 1MCMDADS).
-  - Verify telemetry has returned to normal.
+  - Verify telemetry has returned to normal: Mechanism temperatures are open-circuit, ~440 C;
+    Limit switches are open circuit: OPEN and CLOS. Door position 1DACTAT is 70+/- 5 degrees.
 
+* Contact the GOT Duty Officer to inform that we need the dump data as soon as possible and to
+  email or call us when the dump file is available.
+* Process the dump data and make sure that there is nothing anomalous in the data *BEFORE*
+  the shutdown. We want to know if a new occurrence looks just like the previous occurrences.
```

Comment:
This should probably say "before the anomaly", instead of "shutdown". I'm guessing that this was copied over from one of the shutdown pages.